src/rustup-dist/src/manifest.rs

```diff
@@ -118,6 +123,28 @@ impl Manifest {
         result
     }
 
+    fn table_to_renames(mut table: toml::Table, path: &str) -> Result<HashMap<String, String>> {
+        let mut result = HashMap::new();
+        let rename_table = try!(get_table(&mut table, "rename", path));
+
+        for (k, v) in rename_table {
+            if let toml::Value::Table(mut t) = v {
+                result.insert(k.to_owned(), get_string(&mut t, "to", path)?);
```

Comment:
I was basically following the convention for `pkg`, I also thought it was a little bit more clear, but I'm not attached to the current format, if you'd prefer to change.